src/components/LandingPage/Divider/Divider.component.tsx

```diff
@@ -0,0 +1,49 @@
+import loudspeakerIcon from "@/assets/svg/pillars/loudspeaker.svg";
+import relationsIcon from "@/assets/svg/pillars/relations.svg";
+import targetIcon from "@/assets/svg/pillars/target.svg";
+import Image from "next/image";
+
+type Props = {
+  pillar: string;
+};
+
+export const Divider = ({ pillar }: Props) => {
+  let image, text;
+
+  switch (pillar) {
+    case "relacje":
+      {
+        image = relationsIcon;
+        text = "Relacje";
+      }
+      break;
+    case "glos":
+      {
+        image = loudspeakerIcon;
+        text = "Głos";
+      }
+      break;
+    case "wplyw":
+      {
+        image = targetIcon;
+        text = "Wpływ";
+      }
+      break;
+  }
```

Comment:
Takie podejście nie jest zgodne z zasadami Reacta.

W Reactcie nie możemy tak definiować zmiennych, ponieważ powoduje to negatywny wpływ na optymalizację aplikacji i może być problematyczne przy rerenderach. React przy każdym wykryciu zmian, renderuje komponent (w praktyce: wywołuje funkcję, którą ten komponent jest).

Poprawić sytuację można zrobić to na jeden ze sposobów:

1. Stworzyć funkcje zwracające image i text, tak aby później wyglądało to mniej więcej tak:
```
// jak widać, zwracam z tej funkcji obiekt, a nie dwie różne zmienne
const { image, text } = getImageAndText(pillar);
```

2. Zrobić z image i text zmienne z pomocą `useState`, a następnie zmodyfikować je przy pierwszym renderze z pomocą `useEffect(() => { /* logika zmieniająca stan image/text */ }, [])`

3. Najlepsze rozwiązanie: Użyć metody dictionary, czyli tworzymy obiekt, w którym klucze to nasze przypadki (relacje, glos, wplyw), a wartości to obiekty `{ image: ..., text: .... }`.

Zachęcam do zapytania ChatGPT o implementację, przykład: https://chatgpt.com/share/675247b9-a180-8004-95ca-6915a427cdc0